homeassistant/components/freebox/router.py

```diff
@@ -72,6 +75,7 @@ def __init__(
         self.devices: dict[str, dict[str, Any]] = {}
         self.disks: dict[int, dict[str, Any]] = {}
         self.supports_raid = True
+        self.supports_hosts = True
```

Comment:
at line 75, juste before `self.devices: ...`, as it's linked to it.

Reply:
Done